graylog2-web-interface/src/components/users/UserEdit/TeamsSection.jsx

```diff
@@ -0,0 +1,41 @@
+// @flow strict
+import * as React from 'react';
+import styled from 'styled-components';
+import { PluginStore } from 'graylog-web-plugin/plugin';
+
+import User from 'logic/users/User';
+
+import SectionComponent from '../SectionComponent';
+
+type Props = {
+  user: User,
+};
+
+const Header = styled.h4`
+  margin-bottom: 5px;
+`;
+
+const NoEnterpriseComponent = () => (
```

Comment:
This looks to be the same as `src/components/users/UserDetails/TeamSection.jsx:18`

Can we add a link to `/system/enterprise` so that users can sign up for Enterprise?

Reply:
No because there is not `/system/enterprise` since there is no Enterprise Plugin installed